financial_functions/lambda_handlers.py

```diff
@@ -334,3 +333,23 @@ def sln_handler(request, context):
     args = [request['cost'], request['salvage'], request['life']]
     return __call_ff('sln', args)
 
+def financial_function_generic_handler(request, context):
+    """
+    This function takes in an arbritary financial function and its parameters as inputs and returns the result of that calculation
+    :param request: Dict containing the financial function name and its parameters
+    :param context: Lambda execution context
+    :return: Dict with a 'result' entry containing the result of the calculation
+    """
+    logger.info("financial function request: {}".format(request))
+
+    function_name = request.get("functionName")
```

Comment:
We've been using underscores for request params, so let's be consistent and change to `function_name`. Also, I think this is more pythonic:

```
if 'function_name' not in request:
    return {'error': '...'}
```